test/schedulers.py

```diff
@@ -38,13 +40,15 @@ def __init__(self, optimizer, num_steps_per_epoch, divide_every_n_epochs=20,
   def _epoch(self):
     return self._step_count // self._num_steps_per_epoch
 
+  def _is_warmup_epoch(self):
+    return self._epoch() < math.ceil(self._num_warmup_epochs)
```

Comment:
Hmm, so you change this from 1 to 0.96, but then you do a `ceil()` on it ... what am I missing?

Reply:
Line 52 is where there will be a difference. num_warmup_steps will be slightly different (~1200 instead of ~1251 for batch_size=128), so the slope of the LR graph will be slightly different.

Somehow this little change gets us a ~0.5% accuracy boost, putting us at 76.11% instead of 75.7%